pkg/storage/stores/indexshipper/compactor/deletion/metrics.go

```diff
@@ -81,11 +80,6 @@ func newDeleteRequestsManagerMetrics(r prometheus.Registerer) *deleteRequestsMan
 		Name:      "compactor_pending_delete_requests_count",
 		Help:      "Count of delete requests which are over their cancellation period and have not finished processing yet",
 	})
-	m.deletedLinesTotal = promauto.With(r).NewCounterVec(prometheus.CounterOpts{
```

Comment:
You're removing a metric here which I suppose some folks might be using already? Best to include this in the upgrade guide / changelog.

Reply:
Also, what did you mean by "The metric wasn't always showing the latest value."? Seems like this metric would be pretty useful...

Reply:
Good point. I've left the metric. Instead of creating a new metric it references the main metrics struct now.

Reply:
Thanks @MichelHollands :+1: 
Does that address the "The metric wasn't always showing the latest value" issue mentioned above?

Reply:
Yes, it should. I've added additional checks in the unit tests for this as well now.

Reply:
Magic, thanks